tqdm/_tqdm.py

```diff
@@ -342,7 +353,7 @@ def format_meter(n, total, elapsed, ncols=None, prefix='',
                 bar_length, frac_bar_length = divmod(
                     int(frac * N_BARS * 10), 10)
 
-                bar = '#' * bar_length
+                bar = '█' * bar_length
```

Comment:
this will force an attempt at unicode on systems which do not support it. to force unicode, please use `ascii=False` instead (which also looks better/smoother)